app/views/shared/_flash_notifications.html.erb

```diff
@@ -1,7 +1,8 @@
 <%# this page will display the notifications as a flash message Randal Myers, 22 March 2017 %>
 
 <% course_ids = [] %>
-<% current_user.participants.each{|p| course_ids << p.assignment.course.id if p.assignment and p.assignment.course} %>
+<%#Line 5 implements eager loading using 'includes' method. For detail, visit: http://tiny.cc/9aomuz%>
+<% current_user.participants.includes(:assignment, assignment: [:course]).each{|p| course_ids << p.assignment.course.id if p.assignment and p.assignment.course} %>
```

Comment:
use && instead of and

Reply:
The only change we made in line 5 was to add 

.includes(:assignment, assignment: [:course]) + comment

other than this, all the code is as it was before. I am assuming it should be fine as it was already there in the beta branch before we even started working.